src/Model/Resolver/ResetPassword.php

```diff
@@ -36,12 +37,24 @@ class ResetPassword implements ResolverInterface {
      */
     protected $session;
 
+    /**
+     * @var CustomerRepositoryInterface
+     */
+    private $customerRepository;
```

Comment:
Please don't use the `private` access modifier.
It makes it unable to override or reuse such properties and methods when we extend this class.
Use `protected` instead.